app/models/pair_programming_exercise_feedback.rb

```diff
@@ -12,6 +12,32 @@ class PairProgrammingExerciseFeedback < ApplicationRecord
   scope :intermediate, -> { where.not(normalized_score: 1.00) }
   scope :final, -> { where(normalized_score: 1.00) }
 
+  enum comment: {
+    difficulty_easy: 0,
+    difficulty_some_what_easy: 1,
+    difficulty_ok: 2,
+    difficulty_some_what_difficult: 3,
+    difficult_too_difficult: 4,
+  }, _prefix: true
+
+  enum time: {
+    estimated_time_less_5: 0,
```

Comment:
Then, for those names, the prefix `estimated_time` can be removed. I am still thinking whether we should add the unit `minutes` (or a short form `min`) to the name, just to make the translations obsolete.

Reply:
With the current version, the naming for less than 5 minutes has an additional underscore before the unit, which all other keys are missing. Shouldn't we use the same naming scheme here?